src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultClaimUidExtractor.cs

```diff
@@ -38,6 +38,17 @@ public string ExtractClaimUid(ClaimsIdentity claimsIdentity)
         // Internal for testing
         internal static IEnumerable<string> GetUniqueIdentifierParameters(ClaimsIdentity claimsIdentity)
         {
+            var subClaim = claimsIdentity.FindFirst(
+                claim => string.Equals("sub", claim.Type, StringComparison.Ordinal));
+            if (subClaim != null && !string.IsNullOrEmpty(subClaim.Value))
+            {
+                return new string[]
+                {
+                    "sub",
+                    subClaim.Value
```

Comment:
Well these 2-`string` arrays are run through a SHA256 hash soon after this method returns. Questions:
- Should they be 3-`string` arrays, adding the issuer?
- Use `Issuer` or `OriginalIssuer` from the `Claim`?